doc/integrator/tilegeneration.rst

```diff
@@ -147,7 +147,7 @@ The main thing to do is to:
 
   .. prompt:: bash
 
-     .build/venv/bin/generate_controller --generate_wmts_capabilities
+     .build/venv/bin/generate_controller --capabilities
```

Comment:
=> `--generate-wmts-capabilities`

Reply:
both are ok indeed! but the one in doc is wrong.